src/index.ts

```diff
@@ -1,16 +1,67 @@
-import { typeDefs } from '@typings/apollo'
-import { ApolloServer } from 'apollo-server'
+import {Business} from '@typings/types'
+import { typeDefs as schema } from '@typings/schema'
+import Fastify from 'fastify'
+import mercurius from 'mercurius'
+import { PrismaClient } from '@prisma/client'
 
+const app = Fastify()
+const prisma = new PrismaClient()
 
+const businesses : Pick<Business, 'name' | 'sustainabilityScore' | 'customerScore' | 'profilePicture' | 'profileText'>[] = [
+    {
+      name: "John",
+      sustainabilityScore: 1.078,
+      customerScore: 5.955,
+      profilePicture: "Sample.JPEG",
+      profileText: "John",
+    }
+  ]
 
 const resolvers = {
   Query: {
-    
-  }
+    users: () => {
+      return prisma.user.findMany(
+      )
+    },
+    businesses: () => {
+      return prisma.business.findMany();
+    },
+    comments: () => {
+      return prisma.comment.findMany();
+    },
+    locations: () => {
+      return prisma.location.findMany();
+    },
+    reviews: () => {
+      return prisma.userReview.findMany();
+    },
+  },
 }
 
-const server = new ApolloServer({typeDefs, resolvers})
+app.register(mercurius, {
+  schema,
+  resolvers,
+  context: (request, reply) => {
+    return { prisma }
+  },
+  graphiql: true
```

Comment:
This needs to change depending on `NODE_ENV` - we don't want it available in production.

Reply:
What do you mean by this?